packages/plugin-coverage/src/lib/nx/coverage-paths.unit.test.ts

```diff
@@ -49,6 +51,11 @@ vi.mock('bundle-require', () => ({
       coverageDirectory: 'coverage',
     };
 
+    const JEST_PRESET: JestCoverageConfig & { preset?: string } = {
+      preset: '../../jest.preset.ts',
```

Comment:
❓ What if the LCOV format is not set in preset configuration either?

Reply:
Then the user will get an error regarding non-existant `lcov.info` file. Recursively resolving Jest presets is not trivial and would be overkill for this simple sanity check.